opentelemetry/proto/common/v1/common.proto

```diff
@@ -53,3 +53,10 @@ message StringKeyValue {
   string key = 1;
   string value = 2;
 }
+
+// InstrumentationLibrary is a message representing the instrumentation library informations
+// such as the fully qualified name and version. 
+message InstrumentationLibrary {
+  string name = 1;
+  string version = 2;
```

Comment:
Please specify the format or add a link to the spec.
https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#obtaining-a-tracer is a bit vague on the version format, though.